Tests/secrets_white_list.json

```diff
@@ -256,6 +256,9 @@
     ],
     "_comment": "ENTER URLS WITHOUT http/s, www and path",
     "urls": [
+      "http://ru.wikipedia.org",
+      "https://golang.org",
+      "http://b",
```

Comment:
this could cause a lot of problems since the comparison is by "contains".
in the test, use longer text before the "problematic" letter